src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Container/UnregisterAzureRmRecoveryServicesBackupContainer.cs

```diff
@@ -41,12 +41,14 @@ public override void ExecuteCmdlet()
             {
```

Comment:
This logic appears to be quite specific to the workload. Shouldn't this be in the corresponding provider for a cleaner implementation?